apps/meeting/src/components/Icons/FeaturedLayout.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I think this demo is supposed to be changed from Apache-2.0 to MIT-0 as the repository is under MIT-0

Reply:
Yeah, that's right, I need to fix that. I will push a new version to fix this when I rebase this commit.